usaspending_api/search/migrations/0050_replace_pac_pan_columns_with_program_activities.py

```diff
@@ -0,0 +1,52 @@
+# Generated by Django 3.2.25 on 2024-12-20 01:14
```

Comment:
Can you confirm that this doesn't attempt to generate any additional fields when using Django 4.2?